### DISCUSSION TITLE
E-mail da Python Nordeste - Problema de autenticação

### DISCUSSION THREAD

**Discussion by cecivieira** [apyb/comunidade#74] [Geral]
Boa tarde!

Não estamos conseguindo nos autenticar no e-mail nordeste@python.org.br. Na tela de erro informa que não há autenticação em 2 etapas, no entanto essa autenticação foi estabelecida há alguns meses. O telefone que está cadastrado para o segundo fator de autenticação é o meu.

Cês (APyB) poderiam dá uma olhadinha?

![image](https://user-images.githubusercontent.com/30605862/229599024-d9384cc3-a4ba-4c13-aeac-9afa31bb54a3.png)

**Comment by rougeth** [comment]
Resolvi com a Ceci. Configuramos autenticação de dois passos e criamos uma nova senha.
